reader/components/pdf-outline-sidebar/PDFOutlineSidebar.jsx

```diff
@@ -30,13 +31,26 @@ class PDFOutlineSidebar extends React.PureComponent {
       store: {
         document: { linkService },
       },
+      dispatch,
     } = this.props
 
-    if (!outline || outline.length === 0)
-      return <h5>PDF does not contain any outline</h5>
+    const finalOutline = [
+      ...(outline || []),
+      {
+        onClick: () => {
+          dispatch(scrollToRelatedPapers())
+        },
+        title: 'Related papers',
```

Comment:
Can we rename the section to 'Related papers' too?

Reply:
https://github.com/oacore/reader/pull/89/commits/3b2a94373692b1d658ed28592882a252b0943a27